src/Template/Saefis/xml/e2b.ctp

```diff
@@ -189,15 +193,15 @@
             ?></patientsex>
             <lastmenstrualdateformat/>
             <patientlastmenstrualdate/>
-            <patientmedicalhistorytext><?php echo $saefi['medical_history']; ?></patientmedicalhistorytext>
+            <patientmedicalhistorytext><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$saefi['medical_history']); ?></patientmedicalhistorytext>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate